scrapers_next/ca/people.py

```diff
@@ -17,11 +18,17 @@ def process_item(self, item):
         name = CSS("a").match(item)[2].text_content()
```

Comment:
`name` could also use an update on the selector because it's currently pulling all names as "Committee Membership" since it's pulling the second link that exists in that row